config/deploy.rb

```diff
@@ -6,10 +6,10 @@
 
 set :home_directory, "/home/lyberadmin"
 
-set :branch, 'master'
+#set :branch, 'master'
 
 # Default branch is :master
-# ask :branch, proc { `git rev-parse --abbrev-ref HEAD`.chomp }.call
+ask :branch, proc { `git rev-parse --abbrev-ref HEAD`.chomp }.call
```

Comment:
Shouldn't the default branch be set to master instead of requiring the person doing the deploy to provide a branch name?